public/apps/configuration/app-router.tsx

```diff
@@ -38,31 +38,37 @@ import { TenantList } from './panels/tenant-list/tenant-list';
 import { UserList } from './panels/user-list';
 import { Action, RouteItem, SubAction } from './types';
 import { ResourceType } from '../../../common';
-import { buildHashUrl, buildUrl } from './utils/url-builder';
+import { buildUrl } from './utils/url-builder';
 import { CrossPageToast } from './cross-page-toast';
 import { getDataSourceFromUrl, LocalCluster } from '../../utils/datasource-utils';
+import { getBreadcrumbs } from './utils/resource-utils';
```

Comment:
It looks like this is pulling from inside the security dashboards. How come we are now manually creating the breadcrumbs? Does core no longer offer these utilities?

Reply:
I just moved it into a util function since I needed access to it from the new component - just a refactoring change